### DISCUSSION TITLE
Bolded Text in Quick Reblog tags area

### DISCUSSION THREAD

**Discussion by FishstickPaladin** [AprilSylph/XKit-Rewritten#1473] [Help Desk]
Today, while on desktop I noticed that the Quick Tags extension for XKit Rewritten now has the inputted text bolded. Is there a way to turn off the bolded text, or a reason why this is occurring? Thanks!
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/85412629/714216ca-c369-419f-baa0-ba2b3b6f20fe)

**Comment by marcustyphoon** [comment]
Probably related to #1474! Will take a look.
